content/sensu-go/6.2/sensuctl/_index.md

```diff
@@ -23,16 +23,33 @@ To set up sensuctl, run `sensuctl configure` to log in to sensuctl and connect t
 sensuctl configure
 {{< /code >}}
 
+This starts the prompts for interactive sensuctl setup.
 When prompted, type the [Sensu backend URL][6] and your [Sensu access credentials][8].
 
 {{< code shell >}}
+? Authentication method:
+❯ username/password
+  OIDC
 ? Sensu Backend URL: http://127.0.0.1:8080
 ? Username: YOUR_USERNAME
 ? Password: YOUR_PASSWORD
 ? Namespace: default
 ? Preferred output format: tabular
 {{< /code >}}
 
+If you select the OIDC authentication method, if you are using a desktop, a browser will open to `OIDC provider` and allow you to authenticate and log in.
+If a browser does not open, launch a browser to complete the login via your OIDC provider at following URL:
+   - http://127.0.0.1:8080/api/enterprise/authentication/v2/oidc/authorize
+
+{{< code shell >}}
+? Authentication method:
+  username/password
+❯ OIDC
+? Sensu Backend URL: http://127.0.0.1:8080
+? Namespace: default
+? Preferred output format: tabular
+{{< /code >}}
+
```

Comment:
Again here, I'm assuming the prompts will continue for backend URL, namespaces, and format, but I can correct this if I'm wrong.

Reply:
That's right, we will ask for all information and then open the browser tab.

Here's an actual example of that:

```
$ sensuctl configure
? Authentication method: OIDC
? Sensu Backend URL: http://127.0.0.1:8080
? Namespace: default
? Preferred output format: tabular
Launching browser to complete the login via your OIDC provider at following URL:

	http://127.0.0.1:8080/api/enterprise/authentication/v2/oidc/authorize?callback=http%3A%2F%2Flocalhost%3A8000%2Fcallback

You may also manually open this URL. Waiting for callback...
```